index.html

```diff
@@ -217,5 +217,32 @@ <h4>Skills</h4>
     <hr class="skills">
 </section>
 </main>
+
+<!-- Footer Section -->
+
+<footer class="contact-forms">
+    <div class="contact">
+        <h2 class="form-heading">Contact me</h2> 
```

Comment:
- [ ] OPTIONAL: I strongly recommend you adding more space between the form and about me sections.

- [ ] OPTIONAL: I strongly recommend you adding the background images in the form: 

Your design  | Figma design
------------- | -------------
<img width="226" alt="Screenshot 2022-04-02 at 11 48 36" src="https://user-images.githubusercontent.com/77038610/161372822-14d8f8b8-759c-432e-93e0-2395e9574a33.png">  | <img width="238" alt="Screenshot 2022-04-02 at 11 48 59" src="https://user-images.githubusercontent.com/77038610/161372838-5a5ce6b1-b8ca-408f-9a37-600e9215bf3b.png">